install/operator/pkg/generator/assets/install/04-syndesis-meta.yml.tmpl

```diff
@@ -76,14 +76,14 @@
             valueFrom:
               fieldRef:
                 fieldPath: metadata.namespace
-{{#Debug}}
+{{if .Debug}}
           - name: JAVA_DEBUG
             value: "true"
```

Comment:
I've probably missed it but can you confirm if this debug env var has been bubbled up into the install scripts? I can't see it & if not it would be most useful if a switch could be added to have JAVA_DEBUG turned on upon installation so that devs don't have to manually turn it on after.
WDYT?

Reply:
I've not looked into it either too much.  I was just porting what was existing.  Can you open a new issue to looking to that.

Reply:
Yeah, sure thing. I'm working on the bash scripts atm so can add to my list.

Reply:
How does https://github.com/syndesisio/syndesis/issues/6264 sound?